pkg/gui/main_panel.go

```diff
@@ -22,9 +22,17 @@ func (gui *Gui) scrollDownMain(g *gocui.Gui, v *gocui.View) error {
 		_, sy := mainView.Size()
 		y += sy
 	}
+
+	_, sizeY := mainView.Size()
+	scrollCount := gui.Config.UserConfig.Gui.ScrollHeight
+	totalLines := mainView.ViewLinesHeight()
+	if oy+sizeY >= totalLines {
+		return nil
+	}
+
 	// for some reason we can't work out whether we've hit the bottomq
 	// there is a large discrepancy in the origin's y value and the length of BufferLines
-	return mainView.SetOrigin(ox, oy+gui.Config.UserConfig.Gui.ScrollHeight)
+	return mainView.SetOrigin(ox, oy+scrollCount)
```

Comment:
scrollCount is no longer needed. I will put the original change back.
Same for line 58.

Reply:
Done